syntax = "proto3";

package NullSpaceIPC;


message EffectCommand {
	enum Command {
		UNKNOWN = 0;
		PLAY = 1;
		PLAY_CONTINUOUS = 2;
		HALT = 3;
		PLAY_RTP = 4;
	}
	Command command = 1;
	uint32 area = 2;
	string effect = 3;
	float strength = 4;
}
